Database.py

```diff
@@ -32,58 +43,106 @@ def get_stops_from_function(self,function:str)->list:
 
         return self.cursor.execute(query).fetchall()
 
-    def add_lines_to_json(self,dir):
+    def add_lines_to_json(self, dir: str):
+        """
+        Adds lines to a JSON database. Line = {"name", "stops", "times"}.
+        """
+        used_stops = set()  # used for checking if all stops from the database were used in the process of extraction from html templates
+        lines1 = []
 
-        lines1 =[]
         for files in os.listdir(dir):
```

Comment:
Zakładamy, że we wskazanym directory znajdują się tylko pliki zawierające dane dotyczące linii. Jeślibyśmy jednak założyli że tak nie jest, prosty if sprawdzający jego nazwę zaoszczędziłby wykonywania funkcji na pliku, na którym jest to bezcelowe.
Do przemyślenia czy chcemy zaimplementować taką funkcjonalność